src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -312,29 +312,28 @@ public async Task When_Text_IsWrapping_Set()
 		{
 			var textbox = new TextBox();
 
-			var initialHeight = 500;
-
+			var initialHeight = 500;					
+			
 			textbox.Height = initialHeight;
-
-			var stackPanel = new StackPanel()
-			{
-				Children = { textbox }
-			};
-
-			WindowHelper.WindowContent = stackPanel;
+			
+			WindowHelper.WindowContent = textbox;
 
 			await WindowHelper.WaitForLoaded(textbox);		
 			
 			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
 			textbox.TextWrapping = TextWrapping.Wrap;
+			await WindowHelper.WaitForIdle();
 
-			Android.Graphics.Rect textBounds = new Android.Graphics.Rect();
+			var field = textbox.GetType().GetField("_textBoxView", System.Reflection.BindingFlags.NonPublic
```

Comment:
Avoid reflection whenever possible. In this case, you can find the subview using `FirstFirstChild`, like this:

https://github.com/unoplatform/uno/blob/acc3cab3e2940e9c3ac7845c4d47ffad28389239/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Native_View.cs#L36

and use `TextBoxView` as the type.